lib/msal-browser/src/config/Configuration.ts

```diff
@@ -122,7 +123,8 @@ export function buildConfiguration({ auth: userInputAuth, cache: userInputCache,
         postLogoutRedirectUri: "",
         navigateToLoginRequestUrl: true,
         clientCapabilities: [],
-        protocolMode: ProtocolMode.AAD
+        protocolMode: ProtocolMode.AAD,
+        azureCloudInstance: AzureCloudInstance.None
```

Comment:
I'm still not sure we need a `None` value, as not being defined semantically is the same as `None`.